src/Actions/index.tsx

```diff
@@ -97,38 +100,45 @@ export default class Actions extends React.Component<Props & DefaultProps, State
     return (
       <div style={this.styles.container} ref={this.container}>
         <ActionGroup>
-          <Button
-            data-rh='Default'
-            data-custom-at='right'
-            onClick={() => this.mapZoom.zoomIn()}
-            icon={faPlus}
-            isMobile={isMobile}
-            style={{ borderRight: '2px solid lightgray' }}
-          />
-
-          <Button
-            onClick={() => this.mapZoom.zoomOut()}
-            icon={faMinus}
-            isMobile={isMobile}
-            style={{ borderRight: '2px solid lightgray' }}
-          />
-
-          <Button
-            onClick={() => this.mapZoom.reset()}
-            icon={faUndoAlt}
-            text='Reset Zoom'
-            isMobile={isMobile}
-            style={{ borderRight: '2px solid lightgray' }}
-          />
-
-          <Button
-            onClick={() => this.props.onClearSelection()}
-            icon={faTimesCircle}
-            text={`Clear${isMobile ? '' : ' All'}`}
-            isMobile={isMobile}
-            style={{ borderRight: isMobile ? '2px solid lightgray' : undefined }}
-          />
-
+          {this.props.showControls && (
```

Comment:
I read the ticket to hide _everything_, including the legend. Are we sure we're supposed to keep the legend visible?

Reply:
`showLegend` already exists for that purpose, why should we create an option that duplicates its behaviour?